generators/app/templates/src/app/screens/Login/layout.tsx

```diff
@@ -3,24 +3,30 @@ import i18next from 'i18next';
 
 import FormInput from '~components/FormInput';
 import PATHS from '~components/Routes/paths';
-import RadioGroup from '~components/RadioGroup';
-import RadioOption from '~components/RadioGroup/components/RadioOption';
+import { withSpinner } from '~components/Spinner';
+import { LoginError } from '~services/AuthServices';
+import { Nullable } from '~utils/types';
+import { Error } from '~app/hooks/useRequest';
+import { stringArrayToObject } from '~utils/array';
 
-import { FIELDS } from './constants';
 import styles from './styles.module.scss';
 
 interface Props {
-  onEmailChange: (event: React.FormEvent<Element>) => void;
+  onEmailChange: (event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
   onLogin: (event: React.FormEvent<HTMLFormElement>) => void;
-  onPasswordChange: (event: React.FormEvent<Element>) => void;
+  onPasswordChange: (event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+  loading?: boolean;
+  loginError?: Nullable<Error<LoginError>>;
 }
 
-function Login({ onEmailChange, onPasswordChange, onLogin }: Props) {
+const FIELDS = stringArrayToObject(['email', 'password']);
+function Login({ onEmailChange, onPasswordChange, onLogin, loginError, loading }: Props) {
+  const isError = loginError?.errorData?.message;
   return (
-    <form className={`column center full-width m-top-8 ${styles.formContainer}`} onSubmit={onLogin}>
+    <form className={`column center full-width ${styles.formContainer}`} onSubmit={onLogin}>
       <div className="column center m-bottom-3">
-        <h2 className="m-bottom-1">{i18next.t('Login:login')}</h2>
-        <h3>{i18next.t('Login:loginExplanation')}</h3>
+        <h1 className="m-bottom-1">{i18next.t('Login:login')}</h1>
+        <h2>{i18next.t('Login:loginExplanation')}</h2>
       </div>
       <div className={`column m-bottom-2 ${styles.sectionContainer}`}>
```

Comment:
I think the form could be the section container and the section container could be the form. that way you'll have a div that wraps everything and a for which contains the fileds.